talk/owt/sdk/include/cpp/owt/base/videoframepostprocessor.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) <2021> Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OWT_BASE_VIDEOFRAMEPOSTPROCESSOR_H_
+#define OWT_BASE_VIDEOFRAMEPOSTPROCESSOR_H_
+
+#include <string>
+
+namespace rtc {
+template <typename>
+class scoped_refptr;
+}
+
+namespace webrtc {
+class VideoFrameBuffer;
+}
+
+namespace owt {
+namespace base {
+
+class VideoFramePostProcessor {

```

Comment:
Please add class doc and API doc in doxygen format.

Reply:
What is the coding style of document comments? There are many styles in `talk/owt/sdk/include/cpp`.

Reply:
follow webrtc coding style. https://chromium.googlesource.com/external/webrtc/+/a64685325c2f8f51873b67ae8a91f94ffb19d70b/style-guide.md